app/Foliage/CmdBuild.hs

```diff
@@ -92,8 +92,10 @@ buildAction
 
     makeIndexPage outputDir
 
+    makePreferedVersionsFiles outputDir packageVersions
```

Comment:
Maybe here, rather than writing the actual files (where?) we can instead construct from `packageVersions` something like `Map PackageName [(Timestamp, VersionRange)]`?